users/users_schemas.md

```diff
@@ -0,0 +1,35 @@
+# User schemas
+
+[User List Item](#user-list-item)  
+[User](#user)  
+
+## User List Item
+
+```typescript
+type UserListItem = {
+    id: string
+  ; name: string
+}
+```
+
+## User
+
+```typescript
+type User = {
```

Comment:
schema for type user missing 'department: string' member

Reply:
updated.